frontend/src/services/actual_profile_backend/actual_profile_backend.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Communicates with update 
+ */
+export class ProfileBackendService {
+  
+  loadProfile(id: string): Promise<string> {
```

Comment:
Higher level comment, and I'm sorry I didn't call this out before (this can happen later) but I think having the JSON parsing in here actually makes more sense. So instead of taking a string and returning a string it would take an object (corresponding to the request fields) and return an object (corresponding to the response fields). So basically JSON.stringify() and JSON.parse() or some equivalent would happen inside ProfileBackendService and not inside whoever calls it. This allows for a stronger type interface since you could do something like this:

```ts
interface LoadThingRequest {
  thingId: string;
}

interface LoadThingResponse {
  thing_field: string;
  other_thing_field: OtherInterface;
}

export class ThingBackendService {
  async loadThing(request: LoadThingRequest): Promise<LoadThingResponse> {
    ...
  }
}
```

Reply:
Noted, this is definitely a solid architecture idea and I will begin working on it within a future PR. This also is related to linking up with the backend, so I imagine having discussions with Joey and Lina to clarify what functionality they do and don't have on the servers would be pretty valuable while implementing it. That said, I've made note and will get on it soon!